modules/monitoring-specifying-how-a-service-is-monitored.adoc

```diff
@@ -29,30 +28,28 @@ The `prometheus-example-app` sample service does not support TLS authentication.
 
 .Procedure
 
-. Create a YAML file for the `ServiceMonitor` resource configuration. In this example, the file is called `example-app-service-monitor.yaml`.
+. Create a new YAML configuration file named `example-app-service-monitor.yaml`.
 
-. Add the following `ServiceMonitor` resource configuration details:
+. Add a `ServiceMonitor` resource to the YAML file. The following example creates a service monitor named `prometheus-example-monitor`:
 +
 [source,yaml]
 ----
 apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
-  labels:
-    k8s-app: prometheus-example-monitor
   name: prometheus-example-monitor
-  namespace: ns1
+  namespace: ns1 #<1>
 spec:
   endpoints:
   - interval: 30s
     port: web
     scheme: http
-  selector:
+  selector: #<2>
     matchLabels:
       app: prometheus-example-app
 ----
-+
-This defines a `ServiceMonitor` resource that scrapes the metrics exposed by the `prometheus-example-app` sample service, which includes the `version` metric.
+<1> Specify a user-defined namespace where you want the `ServiceMonitor` resource to run.
+<2> Configure selector to specify the endpoints objects. This example scrapes the metrics exposed by the `prometheus-example-app` service.
```

Comment:
The 2 key points are:
* the selector which matches 1 or many services in the same namespace based on their metadata labels. In almost all cases, you want a selector to match only 1 service. 
* the endpoint's port which tells Prometheus which port to scrape (a service may expose multiple ports).